src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -969,15 +970,29 @@ public UUID runSecurityAnalysis(UUID studyUuid, List<String> contingencyListName
         String receiver;
         try {
             receiver = URLEncoder.encode(objectMapper.writeValueAsString(new NodeReceiver(nodeUuid)),
-                    StandardCharsets.UTF_8);
+                StandardCharsets.UTF_8);
         } catch (JsonProcessingException e) {
             throw new UncheckedIOException(e);
         }
 
         Optional<UUID> prevResultUuidOpt = networkModificationTreeService.getSecurityAnalysisResultUuid(nodeUuid);
         prevResultUuidOpt.ifPresent(securityAnalysisService::deleteSaResult);
 
-        UUID result = securityAnalysisService.runSecurityAnalysis(networkUuid, reportUuid, nodeUuid, variantId, provider, contingencyListNames, parameters, receiver);
+        SecurityAnalysisParameters securityAnalysisParameters = new SecurityAnalysisParameters();
```

Comment:
As we do not manage security analysis parameters persistence in study (not yet), maybe it should be better if no loadflowparameters stored to not send security analysis parameters and let security analysis server decide which are the default one